dialects/datafusion.yaml

```diff
@@ -25,7 +25,31 @@ scalar_functions:
       overflow: ERROR
       rounding: TIE_TO_EVEN
   - name: modulus
-    unsupported: true
```

Comment:
better to represent this as unsupported in all the kernels?

Reply:
what do you mean by unsupported in all kernels?

Reply:
Sorry, I did not understand this earlier. I thought instead of just mentioning it as `unsupported=True`, will it be better, if we specifically write ``unsupported=True` for all the kernels. But, now I think this is the better way.